testapi.pm

```diff
@@ -1806,6 +1806,14 @@ sub diag {
     return bmwqemu::diag(@_);
 }
 
+=head2 host_ip
+    Return VM's host IP
+    in a kvm instance you reach the VM's host under 10.0.2.2
```

Comment:
I am a bit confused here. If we are looking for VM's host, than in case of `svirt` it is not `WORKER_HOSTNAME` but `VIRSH_HOSTNAME`. If you are actually looking for an entry point to worker, then the name `host_ip` does not fit.

Reply:
But asmorodskyi is just extracting the method, right?